### DISCUSSION TITLE
Can we us VS 2019?

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#2697] [Help]
Please forgive this question...

Can we use Visual Studio 2019 for Tailwind development?

If so, is there Intellisense support?

Thank you!

**Comment by eric-hc** [comment]
Yes and yes! https://tailwindcss.com/docs/intellisense

**Reply by ghost** [reply]
@eric-hc these docs are for VS Code, not Visual Studio 2019.  Was there another link?

**Reply by eric-hc** [reply]
Ah sorry, misread that. My mistake! I'm not sure about Studio, but does it let you download? https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss

**Comment by HarlanHugh** [comment]
Seems this can be done by telling VS 2019 where to find the CSS files...
https://gist.github.com/JeffEmery/c2128dc61dfc80249a77d2b59afe72bb#tailwind-css
